02 - Community Expansions/02.01 Plugins by Category/Plugins with custom views.md

```diff
@@ -14,30 +14,30 @@ Plugins that change or add new views, modify how notes are displayed or add popo
-- [[obsidian-gallery|Gallery]]
-- [[obsidian-dice-roller|Dice Roller]]
-- [[obsidian-kanban|Kanban]]
-- [[obsidian-metatable|Metatable]]
-- [[obsidian-2hop-links-plugin|2Hop Links Plugin]]
-- [[obsidian-5e-statblocks|5e Statblocks]]
-- [[obsidian-zoom|Zoom]]
-- [[better-fn|Better footnote]]
-- [[obsidian-habit-tracker|Habit Tracker]]
-- [[obsidian-timeline|Timeline]]
-- [[breadcrumbs|Breadcrumbs]]
-- [[hover-external-link|Hover External Link]]
-- [[obsidian-image-toolkit|Obsidian Image Toolkit]]
-- [[oz-image-plugin|Ozan's Image in Editor Plugin]]
-- [[obsidian-stille|Stille]]
-- [[obsidian-card-view-mode|Card View Mode]]
-- [[obsidian-focus-mode|Focus Mode]]
+- [[obsidian-mind-map|Mind Map]]: A plugin to preview notes as Markmap mind maps
+- [[obsidian-comments|Comments]]: Add, track and easily navigate between a note's Comments
+- [[obsidian-timelines|Timelines]]: Create a timeline view of all notes with the specified combination of tags
+- [[obsidian-daily-stats|Daily Stats]]: Track your daily word count across all notes in your vault.
+- [[obsidian-gallery|Gallery]]: Main Gallery to tag / filter / add notes to images. Display blocks to embed images inside notes. Display block to an image information
+- [[obsidian-dice-roller|Dice Roller]]: Inline dice rolling for Obsidian.md
+- [[obsidian-kanban|Kanban]]: Create markdown-backed Kanban boards in Obsidian.
+- [[obsidian-metatable|Metatable]]: Displays the full frontmatter as a table.
+- [[obsidian-2hop-links-plugin|2Hop Links Plugin]]: Add links to other pages at the bottom of the editor.
+- [[obsidian-5e-statblocks|5e Statblocks]]: Create 5e styled statblocks in Obsidian.md
+- [[obsidian-zoom|Zoom]]: Zoom into heading and lists.
+- [[better-fn|Better footnote]]: Footnote popover for Obsidian
+- [[obsidian-habit-tracker|Habit Tracker]]: This plguin creates a simple Month view for visualizing your punch records.
+- [[obsidian-timeline|Timeline]]: Used to build great timelines
+- [[breadcrumbs|Breadcrumbs]]: Visualise & navigate your vault's structure
+- [[hover-external-link|Hover External Link]]: Hover on external links to see the destination URL.
+- [[obsidian-image-toolkit|Obsidian Image Toolkit]]: This plugin provides some image viewing toolkit.
+- [[oz-image-plugin|Ozan's Image in Editor Plugin]]: View Images, Transclusions, iFrames and PDF Files within the Editor without a necessity to switch to Preview.
+- [[obsidian-stille|Stille]]: Focus on your writing, a section at a time.
+- [[obsidian-card-view-mode|Card View Mode]]: Enable to view your notes as cards.
+- [[obsidian-focus-mode|Focus Mode]]: Add Focus Mode to Obsidian.
 - [[obsidian-fullscreen-plugin|Fullscreen Focus Mode]] 
-- [[obsidian-view-mode-by-frontmatter|Force note view mode by front matter]]
-- [[obsidian-map-view|Map View]]
+- [[obsidian-view-mode-by-frontmatter|Force note view mode by front matter]]: This plugin allows to force the view mode (preview or source) for a note by using front matter
```

Comment:
Line 38 - `obsidian-fullscreen-plugin` does have a description:

> This plugin allows viewing a single document in fullscreen focus mode

Diffing the plugin's note with a neighbour, its content is fine.

Looking at line 38, it has a space after the `]]`, which probably means it failed to match the regex for replacement.

Maybe fix this one by hand?